.github/workflows/verify_consumer_pacts.yml

```diff
@@ -0,0 +1,236 @@
+      pb-event-type:
+        description: 'the Pact Broker event type that triggers this workflow'
+        required: true
+        type: string
+      consumer-name:
+        description: 'the consumer name'
+        required: true
+        type: string
+      consumer-version-number:
+        description: 'the version number of the most recent consumer version associated with the pact content'
+        required: true
+        type: string
+      provider-version-number:
+        description: 'the provider version number for the verification result'
+        required: false
+        type: string
+      consumer-version-tags:
+        description: 'the list of tag names for the most recent consumer version associated with the pact content, separated by ", "'
+        required: true
+        type: string
+      consumer-version-branch:
+        description: 'the name of the branch for most recent consumer version associated with the pact content'
+        required: true
+        type: string
+      provider-version-branch:
+        description: 'the name of the branch for the provider version associated with the verification result'
+        required: false
+        type: string
+      consumer-labels:
+        description: 'the list of labels for the consumer associated with the pact content, separated by ", "'
+        required: false
+        type: string
+      provider-labels:
+        description: 'the list of labels for the provider associated with the pact content, separated by ", "'
+        required: false
+        type: string
+      pact-url:
+        description: 'the "permalink" URL to the newly published pact (the URL specifying the consumer version URL, rather than the "/latest" format'
+        required: true
+        type: string
+
+env:
+  CAN_I_DEPLOY_RUN_NAME: 'can-i-deploy-${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt }}'
+  PACT_BROKER_URL: pact-broker.dsp-eng-tools.broadinstitute.org
+
+jobs:
+
+  bump-check:
+    runs-on: ubuntu-latest
+    outputs:
+      is-bump: ${{ steps.skiptest.outputs.is-bump }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Skip version bump merges
+        id: skiptest
+        uses: ./.github/actions/bump-skip
+        with:
+          event-name: ${{ github.event_name }}
+
+  # We only need a new version tag when this workflow is triggered by opening, updating a PR or PR merge.
+  # When triggered by a Pact Broker webhook, the provider version (GIT hash or release tag)
+  # is already included in the payload, then a new version tag wouldn't be needed.
+  regulated-tag-job:
+    needs: [ bump-check ]
+    if: ${{ needs.bump-check.outputs.is-bump == 'no' }}
+    outputs:
+      new-tag: ${{ steps.tag.outputs.new_tag }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get new tag
+        # https://github.com/DataBiosphere/github-actions/tree/master/actions/bumper
+        uses: databiosphere/github-actions/actions/bumper@bumper-0.0.6
```

Comment:
We use the following patterns to render the tag associated with this workflow run and push this tag along with the verification results to Pact Broker. The new `tag.yml` also employs the latest version of bumper (0.3.0) which fixes a bug related to using `dependabot`. Please update your code to match the pattern.

[https://github.com/DataBiosphere/terra-billing-profile-manager/blob/main/.github/workflows/tag.yml](https://github.com/DataBiosphere/terra-billing-profile-manager/blob/main/.github/workflows/tag.yml)

[https://github.com/DataBiosphere/terra-billing-profile-manager/blob/main/.github/workflows/verify_consumer_pacts.yml](https://github.com/DataBiosphere/terra-billing-profile-manager/blob/main/.github/workflows/verify_consumer_pacts.yml)